apps/root/src/frame/components/navigation/index.tsx

```diff
@@ -0,0 +1,176 @@
+import ActiveSignSelector from '@common/components/active-sign-in-selector';
+import ProfileSelector from '@common/components/profile-selector';
+import { DCA_ROUTE, DCA_CREATE_ROUTE, DCA_POSITIONS_ROUTE, SWAP_ROUTE, TRANSFER_ROUTE } from '@constants/routes';
+import { useAppDispatch } from '@hooks/state';
+import useActiveWallet from '@hooks/useActiveWallet';
+import usePushToHistory from '@hooks/usePushToHistory';
+import { changeRoute } from '@state/tabs/actions';
+import { useCurrentRoute } from '@state/tabs/hooks';
+import React, { useCallback } from 'react';
+import { defineMessage, useIntl } from 'react-intl';
+import {
+  Switch,
+  Navigation as NavigationUI,
+  SectionType,
+  AuditsIcon,
+  BugBountyIcon,
+  DocsIcon,
+  HelpIcon,
+  SupportIcon,
+  IconMenuOption,
+  MoonIcon,
+  SunIcon,
+  WalletIcon,
+  IconMenu,
+} from 'ui-library';
+import ConnectWalletButton from '../connect-wallet';
+import { toggleTheme } from '@state/config/actions';
+import { useThemeMode } from '@state/config/hooks';
+import useSelectedLanguage from '@hooks/useSelectedLanguage';
+import { SUPPORTED_LANGUAGES_STRING, SupportedLanguages } from '@constants/lang';
+import useChangeLanguage from '@hooks/useChangeLanguage';
+
+const helpOptions = [
+  {
+    label: defineMessage({ description: 'audits', defaultMessage: 'Audits' }),
+    icon: <AuditsIcon />,
+    url: 'https://github.com/Mean-Finance/dca-v2-core/tree/main/audits',
+  },
+  {
+    label: defineMessage({ description: 'bugBounty', defaultMessage: 'Bug bounty' }),
+    icon: <BugBountyIcon />,
+    url: 'https://immunefi.com/bounty/meanfinance/',
+  },
+  {
+    label: defineMessage({ description: 'docs', defaultMessage: 'Docs' }),
+    icon: <DocsIcon />,
+    url: 'https://docs.mean.finance',
+  },
+  {
+    label: defineMessage({ description: 'faq', defaultMessage: 'FAQ' }),
+    icon: <HelpIcon />,
+    url: 'https://mean.finance/faq',
+  },
+  {
+    label: defineMessage({ description: 'contact&Support', defaultMessage: 'Contact & Support' }),
+    icon: <SupportIcon />,
+    url: 'http://discord.mean.finance',
+  },
+];
+const Navigation = ({
+  children,
+  isLoading,
+  openNewAccountModal,
+}: React.PropsWithChildren<{ isLoading: boolean; openNewAccountModal: () => void }>) => {
+  const dispatch = useAppDispatch();
+  const pushToHistory = usePushToHistory();
+  const currentRoute = useCurrentRoute();
+  const intl = useIntl();
+  const activeWallet = useActiveWallet();
+  const mode = useThemeMode();
+  const selectedLanguage = useSelectedLanguage();
+  const changeLanguage = useChangeLanguage();
+
+  React.useEffect(() => {
+    if (location.pathname === '/positions') {
+      dispatch(changeRoute('positions'));
```

Comment:
I think we should decouple routes from tabs reducer, using `react-roter-dom` functions. We would simplify every redirect with just one operation. And hooks like `useCurrentRoute could just be replaced by implementing `useLocation`. What do you think?

Reply:
good idea